### DISCUSSION TITLE
intended behavior for unordered lists?

### DISCUSSION THREAD

**Discussion by kdallen1** [nvim-neorg/neorg#398] [Q&A] [answered]
I really love the concept of Neorg.  It's something I need in my life.  However (I think) I'm having problems getting it to work as intended, which is keeping me from using it at the moment.

Two main issues:
1. The '-' character (and seemingly no other characters) resets the indentation.
2. Tables don't show up aligned, not even in the ":h Neorg" file.

Please see the pic below for examples of each.

<img width="451" alt="Screen Shot 2022-03-10 at 12 30 22 PM" src="https://user-images.githubusercontent.com/98832945/157731940-a4ca00c4-b177-44cc-8f09-7ece862e54ed.png">

Am I misunderstanding something here?  If this is not the intended behavior, I'd be happy to upload my init.vim.  However, I've been able to recreate this behavior using the most minimal config possible.  Thanks!

**Comment by vhyrro** [comment, marked as answer]
Heyo! Indentation is currently a *big* weak point of Neorg, and I'll try to get it to a better state as soon as I possibly can (i'm just preoccupied with working on other neorg things rn)! Tables are not truly supported and won't be truly supported for a while, as they're an extension of Neorg's syntax in reality, so don't expect those to be perfect as of now :P

**Reply by kdallen1** [reply]
Dear vhyrro,
Thanks for the really fast answer.  This is good, it means I'll just have to adjust my note-taking style :) Looking forward to future updates!

**Comment by vhyrro** [comment]
Just bumping this to let you know that we've improved our indent engine to be much smarter! It's not perfect yet, but we'll be greatly improving it as time goes on. It's definitely better than what we had before tho :p
